### DISCUSSION TITLE
Menu/Nav like laravels documentation

### DISCUSSION THREAD

**Discussion by ghost** [laravel/framework#37970] [Q&A]
hi,
how do you call the menu/sidebar/nav like laravels documentation?
I do not know, how to google it.

If you click on one topic it shows sub topics, and if I refresh the site/page the topic is still selected and not collapsed.
It saves the position in the menu some how..

I hope someone can help me

**Comment by javicordero** [comment]
You have to use the name attribute on <a> element to reference the id you want to
Like this:
`<a name="your-id" > </a> <h1 id="your-id"></h1>`

**Comment by tpetry** [comment]
It‘s using a hash fragment for anchor links. So any search term including hash and anchor/link/target will get you more information.

**Comment by ghost** [comment]
OK, that is how to jump on one page to different locations. But how do i save the active/collapse/state of a navigation/sidebar?
